bundles/best_practices.rst

```diff
@@ -166,6 +166,72 @@ the ``Tests/`` directory. Tests should follow the following principles:
    A test suite must not contain ``AllTests.php`` scripts, but must rely on the
    existence of a ``phpunit.xml.dist`` file.
 
+Travis
+------
+
+A popular way to test open source bundles is by using `Travis CI`_. A good practice
+is to support at least the two latest LTS versions of Symfony. One should also test
+test latest beta release. Here is a recommended configuration file (``.travis.yml``).
+
+.. code-block:: yaml
+
+    language: php
+    sudo: false
+    cache:
+        directories:
+            - $HOME/.composer/cache/files
+            - $HOME/symfony-bridge/.phpunit
+
+    env:
+        global:
+            - TEST_COMMAND="./vendor/bin/simple-phpunit"
+            - SYMFONY_PHPUNIT_DIR="$HOME/symfony-bridge/.phpunit"
+
+    matrix:
+        fast_finish: true
+        include:
+              # Minimum supported Symfony version with the latest PHP version
+            - env: COMPOSER_FLAGS="--prefer-stable --prefer-lowest" SYMFONY_DEPRECATIONS_HELPER="weak"
+
+              # Test the latest stable release
+            - php: 7.0
+            - php: 7.1
+            - php: 7.2
+              env: COVERAGE=true TEST_COMMAND="./vendor/bin/simple-phpunit --coverage-text"
+
+              # Test LTS versions. This makes sure we do not use Symfony packages with version greater
+              # than 2 or 3 respectively. Read more at https://github.com/symfony/lts
+            - php: 7.2
+              env: DEPENDENCIES="symfony/lts:^2"
+            - php: 7.2
+              env: DEPENDENCIES="symfony/lts:^3"
+
+              # Latest commit to master
+            - env: STABILITY="dev"
+
+        allow_failures:
+              # Dev-master is allowed to fail.
+            - env: STABILITY="dev"
+
+    before_install:
+        - composer require --no-update "symfony/phpunit-bridge:^3.3 || ^4"
+        - if [[ $COVERAGE != true ]]; then phpenv config-rm xdebug.ini || true; fi
+        - if [[ -v $STABILITY ]]; then composer config minimum-stability $STABILITY; fi;
+        - if [[ -v $DEPENDENCIES ]]; then composer require --no-update $DEPENDENCIES; fi;
+
+    install:
+        # To be removed when this issue will be resolved: https://github.com/composer/composer/issues/5355
+        - if [[ "$COMPOSER_FLAGS" == *"--prefer-lowest"* ]]; then composer update --prefer-dist --no-interaction --prefer-stable --quiet; fi
+        - composer update ${COMPOSER_FLAGS} --prefer-dist --no-interaction
+        - vendor/bin/simple-phpunit install
```

Comment:
I would call the PHPUNIT global env defined earlier for that

Reply:
nope. The goal of having the env variable is to be able to change the flags passed to the test command to collect coverage. Using it here would break things.

the current goal of `$TEST_COMMAND` is not to contain the path to PHPUnit. It is to allow altering the command being run in the script.

However, my own suggestion about using a `$PHPUNIT_FLAGS` variable instead would help readability IMO

Reply:
>the current goal of $TEST_COMMAND is not to contain the path to PHPUnit.

My comment assumed the usage of `$PHPUNIT_FLAGS` so it would be only about containing the path to PHPUnit, otherwise yeah I agree with you